apps/web/src/app/(signing)/sign/[token]/page.tsx

```diff
@@ -97,7 +100,13 @@ export default async function SigningPage({ params: { token } }: SigningPageProp
                 <NameField key={field.id} field={field} recipient={recipient} />
               ))
               .with(FieldType.DATE, () => (
-                <DateField key={field.id} field={field} recipient={recipient} />
+                <DateField
+                  key={field.id}
+                  field={field}
+                  recipient={recipient}
+                  dateFormat={documentMeta?.dateFormat}
+                  timezone={documentMeta?.timezone}
```

Comment:
We could also consider doing the coalescing here so we don't have to do anything fancy in the child component.

Reply:
@Mythie You mean something like `documentMeta ?? documentMeta.dateFormat : DEFAULT_VALUE`

Reply:
`documentMeta?.dateFormat ?? DEFAULT_VALUE` yes